src/frontend/src/btc/stores/btc-pending-transactions.store.ts

```diff
@@ -0,0 +1,45 @@
+import type { PendingTransaction } from '$declarations/backend/backend.did';
+import { writable, type Readable } from 'svelte/store';
+
+type Address = string;
+type BtcPendingTransactionsStoreData = Record<Address, Array<PendingTransaction>>;
+
+interface BtcPendingTransactionsStore extends Readable<BtcPendingTransactionsStoreData> {
```

Comment:
Should the data hold in the store contains the information if those are certified or not? extending for example `CertifiedData` and `CertifiedStore`

Reply:
It will always be certified. This endpoint can't be called with a query. The endpoint also prunes outdated pending transactions.

This way I didn't need to implement a background tasks updating them.

I added a comment because it's not obvious. Good point.

Reply:
Instead of a comment, what do you think of using the `CertifiedData` type, excluding `certified` and replacing it with `certified: false` as type. That way even code wise it's specified that it's always certified?

Reply:
Do you mean setting `certified: true` always?

Sure I can do that as well.

Reply:
Done

Reply:
> Do you mean setting certified: true always?

Yes sorry, stupid typo.

Reply:
Cool. So what I meant is also the following:

```
type AlwaysCertifiedData<T> = Omit<CertifiedData<T>, "certified"> & {certified: true};
type BtcPendingTransactionsStoreData = Record<Address, AlwaysCertifiedData<Array<PendingTransaction>>>;
```

Maybe `AlwaysCertifiedData` can even find place in the same module as `CertifiedData`?

Reply:
Ah, sorry, for the confusion. Done!

Reply:
I mean, my bad, I started with a typo ;)